main.py

```diff
@@ -28,14 +29,14 @@
 if args["mute"]:
     audio_cfg.toggle_mute()
 
-pygame.font.init()
+pygame.init()
```

Comment:
@rusty-electron Why has `pygame.font.init()` been changed to `pygame.init()`?

Reply:
I probably left it there when debugging some pyinstaller issues. I'll revert it in the merge commit.